platform_umbrella/apps/common_core/lib/common_core/resources/istio/gateways.ex

```diff
@@ -37,5 +38,6 @@ defmodule CommonCore.Resources.Istio.Gateways do
       "gatewayClassName" => "istio-waypoint",
       "listeners" => [%{"name" => "mesh", "port" => 15_008, "protocol" => "HBONE"}]
     })
+    |> F.require_battery(state, :sso)
```

Comment:
I believe we want `:keycloak` here, since users can install keycloak and select a realm for a traditional service or notebook.